ang/civicase/FileUploader.html

```diff
@@ -0,0 +1,53 @@
+<form class="panel panel-default civicase__panel-transparent-header">
+  <fieldset class="panel-body" ng-disabled="block.check()">
+    <div crm-ui-debug="activity"></div>
+    <div crm-ui-debug="uploader.queue.length"></div>
+    <div class="clearfix civicase__file-upload-container" ng-class="{'civicase__file-upload-container--upload-active': isUploadActive()}">
+      <div class="civicase__file-upload-item civicase__file-upload-box pull-left">
+        <div class="civicase__file-upload-dropzone"
+          nv-file-drop nv-file-over uploader="uploader">
+          <i class="material-icons">file_upload</i>
+          <h3>{{ts('Drop file here')}}</h3>
+          <label for="civicase__file-upload-button">{{ts('or click here to browse')}}</label>
+        </div>
+        <input id="civicase__file-upload-button" class="civicase__file-upload-button" type="file" multiple nv-file-select uploader="uploader" />
+      </div>      
+      <div class="civicase__file-upload-details civicase__file-upload-item pull-right">
+        <div class="form-group">
+          <label for="uploadSubject">{{ts('Activity Subject')}}</label>
+          <input type="text" class="form-control" id="uploadSubject" placeholder="" ng-model="activity.subject">
+        </div>
+        <div class="row">
+          <div class="civicase__file-upload-name col-sm-6 col-md-8"><label>{{ts('Name')}}</label></div>
+          <div class="civicase__file-upload-size col-sm-3 col-md-2"><label>{{ts('Size')}}</label></div>
+          <div class="civicase__file-upload-action col-sm-3 col-md-2"></div>
+        </div>
+
+        <div ng-repeat="item in uploader.queue">
+          <div class="row">
+            <h4 class="civicase__file-upload-name col-sm-6 col-md-8" ng-bind="item.file.name"></h4>
```

Comment:
why do we need the `md`classes?

Reply:
It was existing code and is responsible for setting up the layout. However, I feel that we can get rid of `col-sm` classes.